build-linux32.txt

```diff
@@ -0,0 +1,24 @@
+[binaries]
+c = 'gcc'
+cpp = 'g++'
+ar = 'gcc-ar'
+strip = 'strip'
+
+[properties]
+c_args = ['-m32']
+c_link_args = ['-m32']
+cpp_args = ['-m32']
+cpp_link_args = ['-m32']
+pkg_config_path = ['/usr/lib32/pkgconfig']
```

Comment:
This isn't true on all distros.

Reply:
For weird distros it should be fine to bring your own toolchain file or use good old envvars.